### ISSUE TITLE
Databases intro - white board walkthrough

### ISSUE THREAD

**Issue by shiryz** [foundersandcoders/master-reference#531]
This relates to day 1 of [week 6](https://github.com/foundersandcoders/master-reference/tree/master/coursebook/week-6) which has no description.
 
- [ ] Create a repo with guidelines for mentors.
- [ ] Add desired outline for the intro.

**Comment by jsms90** [comment]
They have asked @stevehopkinson to come in and give the same talk that he did for FAC10. Hoping to record this, so it can be replicated in Nazareth & Gaza, or the content can be adapted into an actual whiteboard walkthrough.

Is there a repo with guidelines for mentors that already exists? (you've ticked the box)

**Comment by yvonne-liu** [comment]
@jsms90 https://github.com/foundersandcoders/intro-to-databases

**Comment by shiryz** [comment]
@jsms90, I ticked the box since I created it.

**Comment by jsms90** [comment]
@shiryz Yeah. Just checking whether the tick was an accident, since there was no link to it.

So what's the deal with this? Is it something being created for Nazareth & Gaza but isn't going to be delivered in London @ConchobarMacNessa @macintoshhelper @pbywater?

**Comment by jsms90** [comment]
@shiryz Just having a look through the PRs. Does "[what is a schema and how to make one]"(https://github.com/foundersandcoders/intro-to-databases/blob/e57d37fac928bf09860b0ceafd55f158c1b7d915/README.md#what-is-a-schema-and-how-to-make-one) section not make the [schema research topic](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/week-6/research-afternoon.md#schemas-and-relationships) slightly redundant?

**Comment by shiryz** [comment]
@jsms90, the examples explain the one to one and one too many etc relations without mentioning that, I can change the title

**Comment by jsms90** [comment]
[Link to @stevehopkinson's slides](https://docs.google.com/presentation/d/1OHoYDQoMkSiJI8X_SEq28z3tkw6MDbHRfHtKDwAphZg/edit#slide=id.p)

@sohilpandya provided his camera & recorded the talk :tada: 

@macintoshhelper Will be editing the video and putting it into [FAC Shared/London/Curriculum](https://drive.google.com/drive/folders/0BzAAKt4fbFbgVmZyZUprZzMtQ28?ths=true)

**Comment by jsms90** [comment]
@shiryz [video of @stevehopkinson's talk](https://drive.google.com/file/d/0B-bY1dUd8QjecWUxOHlyaWtpaVE/view), sent to me by the London social media biz dev task group (@AbdullahChaudhry & @rebecacalvoquintero) :+1:

**Comment by yvonne-liu** [comment]
Thanks so much for this @jsms90 @sohilpandya and of course @stevehopkinson!

**Comment by yvonne-liu** [comment]
@AbdullahChaudhry and @rebecacalvoquintero is the video supposed to be only a minute long?

**Comment by sohilpandya** [comment]
@yvonne-liu I'm waiting for @macintoshhelper to take the footage away from the memory card and put it somewhere. 👍

**Comment by shiryz** [comment]
Record keeping: https://drive.google.com/file/d/0B6Q37hp55l8GMEpXVTRQRWJ4Ykk/view
